scripts/opencontrail-install/ubuntu1404/install.rb

```diff
@@ -0,0 +1,141 @@
+    "#{@ws}/build/packages/python-contrail_#{@pkg_tag}_all.deb",
+    "#{@ws}/build/packages/contrail-config_#{@pkg_tag}_all.deb",
+    "#{@ws}/build/packages/contrail-lib_#{@pkg_tag}_amd64.deb",
+    "#{@ws}/build/packages/contrail-control_#{@pkg_tag}_amd64.deb",
+    "#{@ws}/build/packages/contrail-analytics_#{@pkg_tag}_amd64.deb",
+    "#{@ws}/build/debian/contrail-web-core_#{@pkg_tag}_amd64.deb",
+    "#{@ws}/build/debian/contrail-web-controller_#{@pkg_tag}_amd64.deb",
+    "#{@ws}/build/debian/contrail-setup_#{@pkg_tag}_all.deb",
+    "#{@ws}/build/debian/contrail-nodemgr_#{@pkg_tag}_all.deb",
+    "#{@ws}/build/packages/contrail-utils_#{@pkg_tag}_amd64.deb",
+    "#{@ws}/build/packages/contrail-dns_#{@pkg_tag}_amd64.deb",
+    "#{@ws}/build/packages/ifmap-server_0.3.2-1contrail1_all.deb",
+    "#{@ws}/build/packages/ifmap-python-client_0.1-2_all.deb",
+
+    "#{@ws}/build/debian/contrail-openstack-control_#{@pkg_tag}_all.deb",
+    "#{@ws}/build/debian/contrail-openstack-webui_#{@pkg_tag}_all.deb",
+    "#{@ws}/build/debian/contrail-openstack-analytics_#{@pkg_tag}_all.deb",
+
+     # "#{@ws}/build/debian/contrail-openstack-database_#{@pkg_tag}_all.deb",
+
+#   "#{@ws}/build/debian/contrail-f5_3.0-4100_all.deb",
+#   "#{@ws}/build/debian/contrail-openstack-config_#{@pkg_tag}_all.deb",
+]
+
+@compute_thirdparty_packages = [
+    "#{@ws}/thirdparty/python-pycassa_1.11.0-1contrail2_all.deb",
+    "#{@ws}/thirdparty/python-consistent-hash_1.0-0contrail1_amd64.deb",
+    "#{@ws}/thirdparty/python-backports.ssl-match-hostname_3.4.0.2-1contrail1_all.deb",
+    "#{@ws}/thirdparty/python-certifi_1.0.1-1contrail1_all.deb",
+    "#{@ws}/thirdparty/python-geventhttpclient_1.1.0-1contrail1_amd64.deb",
+    "#{@ws}/thirdparty/python-docker-py_0.6.1-dev_all.deb",
+]
+
+@compute_contrail_packages = [
+    "#{@ws}/build/packages/python-contrail_#{@pkg_tag}_all.deb",
+    "#{@ws}/build/packages/contrail-lib_#{@pkg_tag}_amd64.deb",
+    "#{@ws}/build/debian/contrail-setup_#{@pkg_tag}_all.deb",
+    "#{@ws}/build/packages/contrail-utils_#{@pkg_tag}_amd64.deb",
+    "#{@ws}/build/debian/contrail-nodemgr_#{@pkg_tag}_all.deb",
+    "#{@ws}/build/packages/contrail-vrouter-utils_#{@pkg_tag}_amd64.deb",
+    "#{@ws}/build/packages/contrail-vrouter-dkms_3.0-4100_all.deb",
+    "#{@ws}/build/packages/contrail-vrouter-agent_#{@pkg_tag}_amd64.deb",
+    "#{@ws}/build/packages/python-contrail-vrouter-api_#{@pkg_tag}_all.deb",
+    "#{@ws}/build/packages/python-opencontrail-vrouter-netns_#{@pkg_tag}_amd64.deb",
+    "#{@ws}/build/debian/contrail-vrouter-init_#{@pkg_tag}_all.deb",
+    "#{@ws}/build/debian/contrail-nova-vif_#{@pkg_tag}_all.deb",
+#   "#{@ws}/build/debian/contrail-vrouter-common_#{@pkg_tag}_all.deb",
+
+#   "#{@ws}/build/packages/contrail-vrouter-3.13.0-49-generic_#{@pkg_tag}_all.deb",
+#   "#{@ws}/build/packages/python-vrouter-utils_#{@pkg_tag}_amd64.deb",
+]
+
+# Download and extract contrail and thirdparty rpms
+def download_contrail_software
```

Comment:
At the moment, I have install code (list of packages) separate and the
provision code common for fedora31 and ubuntu1404. For fedora, I built the
RPMs from contrail-vnc-private. To get a similar set of packages, I built
the same for ubuntu and kept them under
github.com/rombie/opencontrail-packages/ubuntu1404/.

My plan is, while this (provisioning and configuration) is all sorted out
(for kubernetes, aurora, etc.), we can get the images come from
opencontrail ppa location
for both ubuntu and fedora (and all other platforms that come in). Do we
already have a plan to build and host fedora21 based RPMs in opencontrail
ppa ?

Regards,
Ananth

On Tue, Apr 21, 2015 at 10:22 AM, Pedro Marques <notifications@github.com>
wrote:

> In scripts/opencontrail-install/ubuntu1404/install.rb
> <https://github.com/Juniper/contrail-kubernetes/pull/5#discussion_r28801013>
> :
>
> > +    "#{@ws}/build/debian/contrail-nodemgr_#{@pkg_tag}_all.deb",
> > +    "#{@ws}/build/packages/contrail-vrouter-utils_#{@pkg_tag}_amd64.deb",
> > +    "#{@ws}/build/packages/contrail-vrouter-dkms_3.0-4100_all.deb",
> > +    "#{@ws}/build/packages/contrail-vrouter-agent_#{@pkg_tag}_amd64.deb",
> > +    "#{@ws}/build/packages/python-contrail-vrouter-api_#{@pkg_tag}_all.deb",
> > +    "#{@ws}/build/packages/python-opencontrail-vrouter-netns_#{@pkg_tag}_amd64.deb",
> > +    "#{@ws}/build/debian/contrail-vrouter-init_#{@pkg_tag}_all.deb",
> > +    "#{@ws}/build/debian/contrail-nova-vif_#{@pkg_tag}_all.deb",
> > +#   "#{@ws}/build/debian/contrail-vrouter-common_#{@pkg_tag}_all.deb",
> > +
> > +#   "#{@ws}/build/packages/contrail-vrouter-3.13.0-49-generic_#{@pkg_tag}_all.deb",
> > +#   "#{@ws}/build/packages/python-vrouter-utils_#{@pkg_tag}_amd64.deb",
> > +]
> > +
> > +# Download and extract contrail and thirdparty rpms
> > +def download_contrail_software
>
> Can these packages come from the PPA, instead ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Juniper/contrail-kubernetes/pull/5/files#r28801013>.
>